test/unit/workDiv/src/WorkDivHelpersTest.cpp

```diff
@@ -86,7 +86,7 @@ TEMPLATE_LIST_TEST_CASE("subDivideGridElems.2D.examples", "[workDiv]", alpaka::t
                 props,
                 false,
                 alpaka::GridBlockExtentSubDivRestrictions::Unrestricted)
-            == WorkDiv{Vec{75, 5}, Vec{4, 128}, Vec{1, 1}});
+            == WorkDiv{Vec{2, 300}, Vec{256, 2}, Vec{1, 1}});
```

Comment:
For the block size, the fast-moving index X should be the larger value. The thread indices are mostly used to access the memory directly. Starting a kernel with 2 threads in x-dimension and 256 in Y-dimension will result slow performance.

Reply:
> I think I need a VC to understand why the results changed. 

The results changed because you told me to change the algorithm up here: https://github.com/alpaka-group/alpaka/pull/1830#discussion_r1033377350. If the new results look bad, then we just change the algorithm back to the old behavior. It is not my intend to change behavior with this PR.

So, revert the last commit again?

Reply:
Last commit reverted after offline discussion.